### DISCUSSION TITLE
Issues with reproducing the results

### DISCUSSION THREAD

**Discussion by Sarah-2021-scu** [facebookresearch/fastMRI#279] [General]
Hello All,

I am trying to reproduce the results of 2020 challenge results using varnet model but unable to read the reconstructed output files.

Thank you for the help.

**Comment by mmuckley** [comment]
Hello @Sarah-2021-scu, could you share some more information on what files you're trying to read? Typically we write the reconstructed output files in HDF5 format. There is a package in Python you can use to read them. You can see more information here:

https://docs.h5py.org/en/latest/quick.html
